content/en/contribute/medic/product-development-process/code-health.md

```diff
@@ -8,30 +8,14 @@ description: >
-3. Grab a ticket from the To Do column and work on it, moving the ticket on the board to communicate what you’re doing. The column is in priority order, but if something looks too big to complete before the end of the week or isn’t sufficiently defined then add a comment to the issue and pick a different issue. Any questions, ask the Senior Director of Technology. 
+A Focused Group is building a feature aligned with a quarterly OKR. They get the feature shipped and may need to wait a week to see results from CHWs. The PM/PO looks to a list of tickets for code health and quickly reviews them with the team. Software developers and QA engineers work on those tickets immediately, tracking progress on the [GitHub board](https://github.com/orgs/medic/projects/134/views/1) like anything else.
 
-### Frequently Asked Questions
+During this time almost all attention, from all engineers on the Focused Group, is on heads-down coding on code health tasks. There may still be some other tasks going on like user interviews or initial measurements of the newly deployed feature.
 
-#### Q: Am I on call?
-
-A: No. This is just a way to schedule who is doing Code Health for a given week. That software engineer works as described above. No waking up at night or being a special point-of-contact for emergencies.
-
-#### Q: Can anyone just add tickets into that Code Health queue and get them coded?
-A: No. This is a Product Team internal queue, and only for us to be responsible about the health of our code and system. The Senior Director of Technology is the primary owner of what is in this queue.
-
-#### Q: What should I do when not doing Code Health?
-A: Keep collaborating with your Focused Group to make progress on the OKRs.
-
-#### Q: What should I do with my Focused Group when I am on Code Health?
-A: Do all the regular activities except coding. Attend the weekly meeting, share your thoughts, keep things moving, etc. It just means that for getting things coded, you’ll pause the Focused Group stuff and pull from the Code Health queue for the week.
-
-#### Q: What about Code Health issues that aren’t finished by the end of the week?
-A: It’s likely that issues will go through some Code Review or AT rounds after your Code Health week is over. Keep working on these issues until they are merged. If the change isn’t ready for Code Review at the end of the week then use your judgement about whether it’s worth finishing it off, or handing it off to the next Code Health assignee.
\ No newline at end of file
+At the end of the week the team realigns around work towards the chosen outcome. Code health work should be merged and the full focus of the team is together with the PM/PO on delivering improvements for users. It may be the case that a code health task cannot yet be merged. If that is the case, other developers on the Focused Group should assist to get it complete so everyone can stay together in what is being worked on.
```

Comment:
```suggestion
At the end of the week, the team realigns around work towards the chosen outcome. Code health work should be merged and the full focus of the team is together with the PM/PO on delivering improvements for users. It may be the case that a code health task cannot yet be merged. If that is the case, other developers on the Focused Working Group should assist to get it complete so everyone can stay together in what is being worked on.
```

Reply:
Thanks for the help on the comma. I'm trying to make an effort to reduce my comma usage and sometimes overdo it in the wrong direction :)